packages/fluentui/react-northstar/src/themes/teams/components/Tree/treeTitleVariables.ts

```diff
@@ -1,13 +1,50 @@
 import { pxToRem } from '../../../../utils';
+import * as _ from 'lodash';
 
 export interface TreeTitleVariables {
   color: string;
+  indicatorColor: string;
+
   padding: string;
+  selectionIndicatorMargin: string;
+  borderColor: string;
+  borderStyle: string;
+  borderRadius: string;
+  borderWidth: string;
+  background: string;
+
+  checkedBackground: string;
+  checkedBorderColor: string;
+  checkedIndicatorColor: string;
+
+  disabledBackground: string;
+  disabledBackgroundChecked: string;
+  disabledBorderColor: string;
+  disabledCheckedIndicatorColor: string;
 }
 
+const defaultValue = 'red';
```

Comment:
Is this cruft? Not sure we want this in the end styles... BTW always use variables in the styles and avoid const like this which cannot be targeted from the theme.